postgresql-migration.cabal

```diff
@@ -12,13 +12,15 @@ copyright:                  2014-2021, Andreas Meingast
 category:                   Database
 build-type:                 Simple
 description:                A PostgreSQL-simple schema migration utility
-tested-with:                GHC==9.2.4, GHC==9.0.1, GHC==8.8.4, GHC==8.6.5, GHC==8.4.4, GHC==9.6.1
-
-extra-source-files:         License
-                            Readme.markdown
+tested-with:                GHC==9.6.1
+                            GHC==9.4.4
+                            GHC==9.2.7
+                            GHC==9.0.2
+                            GHC==8.8.4
+                            GHC==8.6.5
+extra-doc-files:            Readme.markdown
```

Comment:
`cabal check` tell me Readme and Changelog should live under `extra-doc-files`.
Licence shouldn't be added as `extra-source-file` - it's already present under `license-file` above.